src/test/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCauseProjectActionHudsonTest.java

```diff
@@ -42,7 +43,10 @@
     @Test
     public void testShowLastFailureOnProjectPage() throws Exception {
         FreeStyleProject project = createFreeStyleProject();
-        project.getBuildersList().add(new Shell("test $BUILD_NUMBER -eq 2"));
+        final CommandInterpreter = Functions.isWindows()
+            ? new BatchFile("test %BUILD_NUMBER% -eq 2")
```

Comment:
is the test function really available on windows platforms?

Reply:
The test can be skipped by
`assumeThat(not(Functions.isWindows()))` or `assumeThat(Functions.isUnix())` in the beginning of the test method.